### PR TITLE
Extend mapcss tags with Detail/Trap/Fix/Resource messages

### PR DESCRIPTION
**Pull request by Famlam**
See #1598

Adds:
`-osmoseDetail`
`-osmoseTrap`
`-osmoseFix`
`-osmoseResource`

Updating actual rules is for after this PR is finished+merged

### CODE REVIEW

File: doc/2-PluginMapCSS.md [hunk 1]
```diff
@@ -206,23 +206,41 @@ A free text on how to fix can also be present:
...
+    -osmoseDetail: "message containing extra details about the issue";
+    -osmoseFix: "message containing a description of how to fix an issue";
+    -osmoseTrap: "message warning about potential mistakes";
+```
+These are purely textual and will not affect any auto-fix. They support translations, placeholders and simple mark-up. For example:
+```css
+    -osmoseFix: tr("Create two separate objects, one with `{0}` and one with `{1}`", "highway=*", "building=*");
+```
+
+Lastly, one can add a link to the OpenStreetMap wiki using the extension `-osmoseResource`:
```
Comment (frodrigo) [inline doc/2-PluginMapCSS.md:237]: A link to anything. Not just the wiki.

File: doc/2-PluginMapCSS.md [hunk 2]
```diff
@@ -206,23 +206,41 @@ A free text on how to fix can also be present:
...
-Set the Osmose-QA class definition fields: `item`, `class` id, and `level`. When defined, override the default level of `throw`.
+Set the Osmose-QA class definition fields: `item`, `class` id, and `level`. When defined, it overrides the default level of `throw*`.
 
-If required a sub class id can also be defined with `[class]:[subclass]`:
+If required, a sub class id can also be defined with `[class]:[subclass]`:
 ```css
     -osmoseItemClassLevel: "4030/40301:887554/2";
 ```
 
+Additionally, the following three properties can be used to give extra information regarding an issue:
```
Comment (frodrigo) [inline doc/2-PluginMapCSS.md:226]: May add a comment about the fact that text could be markdown.

**Comment by Famlam** [comment]
Thanks!

**Comment by frodrigo** [comment]
Thank you for implementing this feature!

**Comment by Famlam** [comment]
I actually just realize I forgot to implement a mapcss equivalent of `example` 🤦

**Pull request merged**